flow360/component/flow360_params/flow360_params.py

```diff
@@ -506,6 +514,7 @@ def to_solver(self, params: Flow360Params, **kwargs) -> Geometry:
     # pylint: disable=missing-class-docstring,too-few-public-methods
     class Config(Flow360BaseModel.Config):
         allow_but_remove = ["meshName", "endianness"]
+        exclude_on_flow360_export = ["mesh_unit"]
```

Comment:
`mesh_unit` exclude is implemented in to_solver() method. It is a duplication of logic here.

Reply:
Good catch.